prompts/configs.go

```diff
@@ -79,6 +79,26 @@ func PromptForTargetConfig(targetName string, target *workflow.Target, existingC
 			Value(&baseServerURL))
 	}
 
+	var docsLanguages []string
+	if target.Target == "docs" {
```

Comment:
This looks like the same-ish codeblock from the other pr?

Reply:
It's sort of the same. You have to unmarshal each item in an []interface{}